core/src/main/java/org/apache/accumulo/testing/core/TestEnv.java

```diff
@@ -96,15 +97,22 @@ public String getPid() {
   }
 
   public Configuration getHadoopConfiguration() {
-    Configuration config = new Configuration();
-    config.set("mapreduce.framework.name", "yarn");
-    // Setting below are required due to bundled jar breaking default
-    // config.
-    // See
-    // http://stackoverflow.com/questions/17265002/hadoop-no-filesystem-for-scheme-file
-    config.set("fs.hdfs.impl", org.apache.hadoop.hdfs.DistributedFileSystem.class.getName());
-    config.set("fs.file.impl", org.apache.hadoop.fs.LocalFileSystem.class.getName());
-    return config;
+    if (hadoopConfig == null) {
+      String hadoopPrefix = System.getenv("HADOOP_PREFIX");
+      if (hadoopPrefix == null || hadoopPrefix.isEmpty()) {
+        throw new IllegalArgumentException("HADOOP_PREFIX must be sent in env");
+      }
+      hadoopConfig = new Configuration();
+      hadoopConfig.addResource(new Path(hadoopPrefix + "/etc/hadoop/core-site.xml"));
```

Comment:
I pushed another commit 62e91527 where I created new properties in `accumulo-testing.properties` to avoid relying on loading Hadoop config files using `HADOOP_PREFIX`.  I also added `accumulo` to several properties that configured Accumulo scanners.